ppp/iteration_2_24102022.md

```diff
@@ -0,0 +1,23 @@
+### Plans 📆
+
+- Reiterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
+- Investigating ldk integration to the flutter mobile app. A little bit prototyping. 🚀
+- Write up a post about adding dlc to lightning channels. 💪
+
+### Progress ✅
```

Comment:
❓ Can we use hyperlinks rather than explicit URLs?

Reply:
I don't think so, but I can't test unfortunately as I can't find the write story button on bolt.fun anymore 🙈